### ISSUE TITLE
Categorizing plots - Venn diagram

### ISSUE THREAD

**Issue by apierr** [apierr/BCDash#22]
Venn diagram about plots

![image](https://user-images.githubusercontent.com/1194257/53892505-b28f8680-402c-11e9-8aa7-e782822013b1.png)

**Comment by Ducasse** [comment]
The Bubble is nice too and it can be for the comparison as well as the scatter plot
I like the many variables too (as soon as you have matrix like dependencies between elements) 
you need it. 

Did you check a bit the roassal book?

**Comment by apierr** [comment]
@Ducasse I read it 4 months ago and I should read it again. By the way, are there some chapters that you recommend to read?
I fixed the picture following your suggestions.

**Comment by Ducasse** [comment]
Not really. I was thinking also to browse the examples in the package to see the visualisation.
